ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/General.hs

```diff
@@ -850,27 +850,26 @@ prop_general_internal syncity pga testOutput =
   Final chains properties
 -------------------------------------------------------------------------------}
 
--- | What was the most number of blocks needed to be dropped from a final chain
```

Comment:
This function was written in the context of `Cardano.hs` but was moved to the more general `General.hs` prematurely. This diff makes it actually generally enough to live in this module.